### PR TITLE
feat(EditorView): add padding variables

### PR DESCRIPTION
**Pull request by whiteformed**
add padding variables for toolbar and editor content
add custom css variables story and documentation

### CODE REVIEW

File: src/bundle/WysiwygEditorView.scss [hunk 1]
```diff
@@ -4,11 +4,13 @@
 
     &__toolbar {
         flex-shrink: 0;
+        padding: var(--g-md-toolbar-padding);
     }
 
     &__editor {
         overflow-y: auto;
         flex-grow: 1;
+        padding: var(--g-md-editor-extra-padding);
```
Comment (makhnatkin) [inline src/bundle/WysiwygEditorView.scss:13]: why extra?

Comment (whiteformed) [reply src/bundle/WysiwygEditorView.scss:13]: > why extra?

cause this adds up to the padding of __editor's class nested elements

File: src/bundle/sticky/sticky.scss [hunk 2]
```diff
@@ -13,6 +13,7 @@ $block: 'g-md-editor-sticky';
     }
 
     &_sticky-active:not(.#{$block}_clear) {
+        padding: 0;
```
[the remark this answers is not in the document]
Comment (whiteformed) [reply src/bundle/sticky/sticky.scss:16]: > Let's use another solution.

added variable for sticky toolbar inset

File: src/bundle/sticky/sticky.scss [hunk 3]
```diff
@@ -15,9 +15,11 @@ $block: 'g-md-editor-sticky';
     &_sticky-active:not(.#{$block}_clear) {
         z-index: zIndexVars.$sticky-toolbar;
 
+        padding: 0;
```
Comment (makhnatkin) [inline src/bundle/sticky/sticky.scss:18]: `--g-md-toolbar-sticky-padding`

**Review by makhnatkin — APPROVED** [review]

**Review by d3m1d0v — APPROVED** [review]

**Pull request merged**